### DISCUSSION TITLE
how does one change the font in an app?

### DISCUSSION THREAD

**Discussion by localdevjs** [atom/atom#22239] [Q&A]
I have several electron apps on linux, the font used in these apps (slack, signal etc) does not respect my desktop (ubuntu, kde or gnome or xfce) font which is currently DejaVu Sans. How do I tell those apps to use my desktop font instead of whatever font it's using right now which is very ugly with anti-aliasing turned off?

**Comment by ThatXliner** [comment]
[Settings](https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences) > Editor > Font-family.

See also: https://flight-manual.atom.io/using-atom/sections/basic-customization/#configuration-key-reference, https://flight-manual.atom.io/getting-started/sections/atom-basics/
